build_deploy.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-# This is a placeholder file needed by the CI/CD templates
-# You can safely ignore this file :)
-exit 0
```

Comment:
We need this file for AppSRE.
Eventually for exactly that reason I'd say we move to our internal AppSRE version of this repo to not have this stuff confuse people in the public. :expressionless:

Reply:
I mean... no comment here. (: